packages/osd-stylelint-plugin-stylelint/src/utils/get_rules_from_config.ts

```diff
@@ -13,15 +13,21 @@ import path from 'path';
 import { readFileSync } from 'fs';
 import { matches } from './matches';
 
+export type FileBasedConfig = Record<string, { approved?: string[] }>;
```

Comment:
Nice! Probably can add this to some top level documentation about the two types of supported configs